### DISCUSSION TITLE
How to migrate data from V3.0.3.8 to V4.0.0.0

### DISCUSSION THREAD

**Discussion by ghost** [opencart/opencart#11365] [Q&A]
Hello,
We have some stores running on V3.0.3.8 and we will be migrating/upgrading them to V4.0.0.0 when some bugs are fixed.

We are interested how would you migrate the users and transactions, statistics of V3.0.3.8. We can regenerate everything else on the new installation.

Thanks!

**Comment by PhamToan** [comment]
Hello, 
There are a lot of change in database. If  you really want to do, you can hire my team https://mmosolution.com
We will take care mirage for you
